src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource.cs

```diff
@@ -79,8 +79,9 @@ private sealed partial class TagSource : ForegroundThreadAffinitizedObject
 
             #region Fields that can only be accessed from the foreground thread
 
-            private readonly ITextView _textViewOpt;
+            private readonly ITextView? _textView;
             private readonly ITextBuffer _subjectBuffer;
+            private readonly ITextBufferVisibilityTracker? _visibilityTracker;
```

Comment:
i presume the actual usage of this is in the followup pr

Reply:
Ooops! this was meant to be used.

Reply:
Ok.  Updated to use this.